setup.py

```diff
@@ -8,7 +8,7 @@
 
 setuptools.setup(
     name='jupyterlab_git',
-    version='0.5.0',
```

Comment:
Please fix the versioning. I'd stay on 0.5 till we release 0.6

Reply:
I'm sorry, I was sure I removed this. Will do now.